openstudiocore/src/utilities/bcl/templates/common/LICENSE.md

```diff
@@ -0,0 +1 @@
+Insert your license here
```

Comment:
Currently we are case sensitive for LICENSE.md and README.md (just like we are for measure.rb)